emscripten.py

```diff
@@ -1798,18 +1798,27 @@ def create_fp_accessors(metadata):
                  'perhaps a side module was not linked in? if this symbol was expected to arrive '
                  'from a system library, try to build the MAIN_MODULE with '
                  'EMCC_FORCE_STDLIBS=XX in the environment");')
+    # the name of the original function. this is generally the normal function
+    # name, unless it is legalized, in which case the export is the legalized
+    # version, and the original provided by orig$X
+    original = mangled[1:]
```

Comment:
The original is already called `name` here I think.. it was just mangled to `mangled` above, no?